fuzzing/config/config.go

```diff
@@ -106,13 +106,28 @@ type TestingConfig struct {
 	PropertyTesting PropertyTestConfig `json:"propertyTesting"`
 }
 
+type AssertionModesConfig struct {
+	FailOnAssertFailed                  bool `json:"fail_on_assert_failed"`
+	FailOnArithmeticOverflow            bool `json:"fail_arithmetic_overflow"`
+	FailOnDivideByZero                  bool `json:"fail_divide_by_zero"`
+	FailOnEnumTypeConversionOutOfBounds bool `json:"fail_on_enum_type_conversion_out_of_bounds"`
+	FailOnIncorrectStorageAccess        bool `json:"fail_on_incorrect_storage_access"`
+	FailOnPopEmptyArray                 bool `json:"fail_on_pop_empty_array"`
+	FailOnOutOfBoundsArrayAccess        bool `json:"fail_on_out_of_bounds_array_access"`
+	FailOnAllocateTooMuchMemory         bool `json:"fail_on_allocate_too_much_memory"`
+	FailOnCallUninitializedVariable     bool `json:"fail_on_call_uninitialized_variable"`
+}
```

Comment:
Can you add a comment for the top-level `AssertionModesConfig` as well as all its parameters? You can use the various structs in the config files as an example.